packages/cli/src/lib/cmds/play.ts

```diff
@@ -68,7 +68,7 @@ class Play {
   async publish(twaManifest: TwaManifest): Promise<boolean> {
     // Validate that the publish value is listed in the available Tracks.
     // If no value was supplied with publish we make it internal.
-    const userSelectedTrack = asPlayStoreTrack(this.args.publish?.toLowerCase() || 'internal');
+    const userSelectedTrack = asPlayStoreTrack(this.args.track?.toLowerCase() || 'internal');
```

Comment:
Is there a fixed list of values that can be passed as a track? Or can any string be used?

Reply:
Its a list of values. They can be found [here](https://github.com/GoogleChromeLabs/bubblewrap/blob/069e7444cf8c71c6ba4f844eff731c3eb97726bc/packages/core/src/lib/GooglePlay.ts#L21)